exampleCourse/questions/element/orderBlocks/question.html

```diff
@@ -0,0 +1,107 @@
+
+</pl-question-panel>
+<pl-order-blocks answers-name="prime-numbers-ordered" min-incorrect="2" max-incorrect="4">
+  <pl-answer correct="false">1</pl-answer>
+  <pl-answer correct="true">2</pl-answer>
+  <pl-answer correct="true">3</pl-answer>
+  <pl-answer correct="false">4</pl-answer>
+  <pl-answer correct="true">5</pl-answer>
+  <pl-answer correct="false">6</pl-answer>
+  <pl-answer correct="true">7</pl-answer>
+  <pl-answer correct="false">8</pl-answer>
+  <pl-answer correct="false">9</pl-answer>
+  <pl-answer correct="false">10</pl-answer>
+</pl-order-blocks>
+
+<pl-question-panel>
+<p>
+  <strong>Example 3:</strong>  Implement the Python function <code>my_sum</code>, which takes 2 numbers, <code>first</code> and <code>second</code>,
+  and return their sum. Note that the ordering of instantiation, computation, and returning of the value must be correct.
+  Remember to indent your code! </p>
+<p>
+ This examples uses <code>grading-method="ranking"</code> to control how the correct solution is ordered
+  and <code>indent</code> attribute in <code>pl-answer</code> to define expected code indentation.
+</p>
+</pl-question-panel>
+
+
+<pl-order-blocks answers-name="python-ordered" grading-method="ranking" indentation="true">
+
+  <pl-answer correct="true" ranking="1" indent="0">def my_sum(first, second):</pl-answer>
+  <pl-answer correct="true" ranking="2" indent="1">sum = 0</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="1">sum += first</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="1">sum += second</pl-answer>
+  <pl-answer correct="true" ranking="4" indent="1">return sum</pl-answer>
+  <pl-answer correct="false">sum + first</pl-answer>
+  <pl-answer correct="false">sum + second</pl-answer>
+  <pl-answer correct="false">This is an example very long tile to test CSS overflow</pl-answer>
+
+</pl-order-blocks>
+
+<hr>
+
+<pl-question-panel>
+<p>
+  <strong>Example 4:</strong> Let $a, b$ be any even numbers. Prove that $a + b$ is always even.
+</p>
+
+<p>This example uses the attribute <code>grading-method="ranking"</code> to control how the correct solution is ordered.
+When the order of a correct block is not relevant,
+ the attribute <code>ranking</code> is not provided. In this example, two of the blocks can be placed at any
+position when constructing the solution. The attribute <code>solution-placement="bottom"</code> is used
+to display the input area above the output area.</p>
+</pl-question-panel>
+
+ <pl-order-blocks answers-name="latex_answers" grading-method="ranking" solution-placement="bottom">
+  <pl-answer correct="true" ranking="1" >Proof by Contrapositive:</pl-answer>
+  <pl-answer correct="true" ranking="2" >Take some $m \in \mathbb{Z}$ </pl-answer>
+  <pl-answer correct="true" ranking="2" >Take some $n \in \mathbb{Z}$ </pl-answer>
+  <pl-answer correct="true" >Assume $m$ is even</pl-answer>
+  <pl-answer correct="true" >Assume $n$ is even</pl-answer>
```

Comment:
This seems a little broken....is there a reason theses two lines don't have a ranking specified?

Regardless, in this proof example, the ranking grader won't correctly grade all cases, you need the DAG based grader. Can we just leave a note in here that it will be updated with a proof-specific grader soon?

Reply:
Yes, I think it was Matt who identified the need for a DAG, and that someone from UIUC has implemented DAG for grading proofs. I'll update the #3476 with the note.